deployment/ansible/roles/driver.monit/tasks/main.yml

```diff
@@ -0,0 +1,49 @@
+---
+- name: Download monit
+  get_url: >
+    url=https://mmonit.com/monit/dist/binary/{{ monit_version }}/monit-{{ monit_version }}-linux-x64.tar.gz
```

Comment:
The version in the Ubuntu repositories doesn't allow passing parameters inside of `check program with path`, which was introduced in 5.7. So for the docker containers that get checked with `docker top <container-name>` we would have had to create a separate wrapper script for checking each docker container with the container name prepopulated.